cmd/root.go

```diff
@@ -43,10 +52,14 @@ func (c *rootContext) getValue(path string) (interface{}, error) {
 
 	var config map[interface{}]interface{}
 	var err error
+	var stream *os.File
```

Comment:
Yeah, pretty sure you want to keep this flexible and use `io.Reader`

Reply:
[Fixed](https://github.com/pastdev/clconf/pull/16/commits/93a001f6441aa3013080a3d152dff92ae24058e8)